modules/core/04-channel/keeper/packet_test.go

```diff
@@ -830,6 +854,25 @@ func (suite *KeeperTestSuite) TestAcknowledgePacket() {
 			suite.Require().NoError(err)
 			channelCap = suite.chainA.GetChannelCapability(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 		}, false},
+		{"channel in tryupgrade and flush status not in flush", func() {
```

Comment:
what does this mean? The chain is in ackupgrade? And the flush status is? (I think flush complete)

Reply:
ta, this should be `"channel in ackupgrade and flush status in flush complete"`